src/Trakx.Circle.ApiClient.Tests/Integration/BankAccountClientTest.cs

```diff
@@ -0,0 +1,104 @@
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+using Xunit.Abstractions;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+
+namespace Trakx.Circle.ApiClient.Tests.Integration;
+
+/// <summary>
+/// Bank account client test
+/// <see cref="BankAccountClientTest"/>
+/// </summary>
+public class BankAccountClientTest : CircleClientTestsBase
+{
+    private readonly IBankAccountsClient _bankAccountsClient;
+    private readonly IPaymentsClient _paymentsClient;
+    private readonly MockCreators _mockCreator;
+    /// <summary>
+    /// constructor for <see cref="BankAccountClientTest"/>
+    /// </summary>
+    /// <param name="apiFixture"></param>
+    /// <param name="output"></param>
+    public BankAccountClientTest(CircleApiFixture apiFixture, ITestOutputHelper output) : base(apiFixture, output)
+    {
+        _bankAccountsClient = ServiceProvider.GetRequiredService<IBankAccountsClient>();
+        _paymentsClient = ServiceProvider.GetRequiredService<IPaymentsClient>();
+        _mockCreator = new MockCreators(output);
+        Logger.Information($"Instantiating  {nameof(BankAccountClientTest)}");
+    }
+
+    /// <summary>
+    /// creating US bank account with valid payload should be successful
+    /// <remarks>
+    /// The documentation state that the response code should be 201, but the sandbox response returned 200
+    /// either 200 or 201 should be valid
```

Comment:
Maybe you can try to use `IsSuccessStatusCode` method from the [HttpResponseMessage](https://msdn.microsoft.com/en-us/library/vstudio/system.net.http.httpresponsemessage(v=vs.118).aspx) class if that is what you are getting back?